src/app/navbar/expandable-navbar-section/expandable-navbar-section.component.spec.ts

```diff
@@ -76,6 +77,78 @@ describe('ExpandableNavbarSectionComponent', () => {
       });
     });
 
+    describe('when Enter key is pressed on section header (while inactive)', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Baryonsoft/dspace-admin/pullRequest?prid=8206188)
```suggestion
    describe("when Enter key is pressed on section header (while inactive)", () => {
```